component-playground/config.js

```diff
@@ -11,7 +11,17 @@ var resolvePath = function(userPath) {
   return path.resolve(cwd, userPath);
 };
 
-module.exports = {
+var config;
+
+try {
```

Comment:
Is there no reliable (sync) way to check if a file exists? I for one try to use _try catch_ as a last resort.

After doing a couple of searches on the topic, I noticed Node is deprecating most sync functions for check if a file exists, but maybe you have more recent experience with Node...